lib/utils/createFromTemplate.js

```diff
@@ -45,16 +46,16 @@ module.exports = (templateName, destPath, options = {}) =>
           BbPromise.all([
             new BbPromise((gitignoreRenameResolve, gitIgnoreRenameReject) => {
               const gitignorePath = join(destPath, 'gitignore');
-              exists(gitignorePath, (hasGitignore) => {
-                if (!hasGitignore) {
-                  gitignoreRenameResolve();
-                  return;
-                }
-                rename(gitignorePath, join(destPath, '.gitignore'), (renameError) => {
-                  if (renameError) gitIgnoreRenameReject(renameError);
-                  else gitignoreRenameResolve();
-                });
-              });
+
+              fs.promises
+                .access(gitignorePath)
+                .then(() => {
+                  rename(gitignorePath, join(destPath, '.gitignore'), (renameError) => {
+                    if (renameError) gitIgnoreRenameReject(renameError);
+                    else gitignoreRenameResolve();
+                  });
+                })
+                .catch(gitignoreRenameResolve);
```

Comment:
Let's replace this whole `new BbPromise` construct with:

```javascript
fs.promises.access(gitignorePath).then(
  () => rename(gitignorePath, join(destPath, '.gitignore')),
  () => {}
)
```

Reply:
got it